packages/cli/src/transforms/transform.html.js

```diff
@@ -33,7 +33,7 @@ class HTMLTransform extends TransformInterface {
         let body = await getAppTemplate(barePath, this.workspace);
         body = await getAppTemplateScripts(body, this.workspace);
         body = getUserScripts(body, this.workspace);
-        body = getMetaContent(url, this.config, body);
+        body = getMetaContent(url.replace('index.html', ''), this.config, body);
```

Comment:
this was sending URLs w/ _index.html_ in the `url` so this strips that to keep nice clean URLs, e.g. 

www.example.com/about/index.html 

is now 

www.example.com/about/

----

although this makes more sense to do in `getMetaContent`?

Reply:
edit x2: actually, looks like I don't need it there, the tests pass without it, so I must have fixed the issue that caused this.  😎